### PR TITLE
feat(ActionBar): vertical mode, selected item icon if not visible

### PR DESCRIPTION
**Pull request by marcinsawicki**
Resolves: #882 

## Description
Vertical mode for component.
Improved selected item indicator, from now it indicates the element selected from the menu with icon visible.
Also changed the `ActionMenu` logic and added the animation on menu show.

## Storybook

https://feature-882--613a8e945a5665003a05113b.chromatic.com/?path=/docs/experimental-actionbar--docs

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [x] Assign pull request with the correct issue

### CODE REVIEW

File: packages/react-components/src/components/ActionBar/ActionBar.module.scss [hunk 1]
```diff
@@ -44,6 +51,108 @@ $base-class: 'action-bar';
 
   &__menu-wrapper {
     display: flex;
-    align-items: center;
+    flex: 0 0 70px;
+    padding: var(--spacing-2) var(--spacing-1);
+
+    &__button {
+      padding: 6px;
```
Comment (mreszke) [inline packages/react-components/src/components/ActionBar/ActionBar.module.scss:58]: spacing token?

File: packages/react-components/src/components/ActionBar/ActionBar.module.scss [hunk 2]
```diff
@@ -44,6 +51,108 @@ $base-class: 'action-bar';
...
+      padding: var(--spacing-1) var(--spacing-2);
+      height: 70px;
+
+      &__button {
+        flex-direction: column;
+        height: auto;
+        min-height: 36px;
+
+        > span {
+          margin-top: 4px;
```
Comment (mreszke) [inline packages/react-components/src/components/ActionBar/ActionBar.module.scss:121]: as above

File: packages/react-components/src/components/ActionBar/types.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+   * Set 'scroll' to disable menu and enable scroll
+   */
+  type?: 'menu' | 'scroll';
+  /**
+   * Set the bar buttons verticaly
+   */
+  vertical?: boolean;
+}
+
+export type IActionBarOption = {
```
Comment (mreszke) [inline packages/react-components/src/components/ActionBar/types.ts:30]: why some times using `type` and sometimes `interface`?

**Review by mreszke — APPROVED** [review]

File: packages/react-components/src/components/ActionBar/ActionBar.module.scss (hunk 1 above)
Comment (marcinsawicki) [reply packages/react-components/src/components/ActionBar/ActionBar.module.scss:58]: This is a value closely related to this component and its logic.

File: packages/react-components/src/components/ActionBar/ActionBar.module.scss (hunk 2 above)
Comment (marcinsawicki) [reply packages/react-components/src/components/ActionBar/ActionBar.module.scss:121]: 👍

File: packages/react-components/src/components/ActionBar/types.ts (hunk 3 above)
Comment (marcinsawicki) [reply packages/react-components/src/components/ActionBar/types.ts:30]: It should be interface here, I'll change that.

**Review by vladko-uxds — APPROVED** [review]

**Pull request merged**